### PR TITLE
Background_processing module

### PR DESCRIPTION
**Pull request by vmayoral**

### CODE REVIEW

**Comment by mlautman** [comment]
Looks good to me!

I would like to see if we could get CI running on this before merging so that we can verify the changes but I would also prefer to not let PR's sit for too long while we wait.

File: moveit_core/background_processing/src/background_processing.cpp [hunk 1]
```diff
@@ -35,12 +35,15 @@
...
 #include <moveit/background_processing/background_processing.h>
-#include <ros/console.h>
+#include "rclcpp/rclcpp.hpp"
 
 namespace moveit
 {
 namespace tools
 {
+// Logger
+rclcpp::Logger LOGGER = rclcpp::get_logger("background_processing");
```
Comment (henningkayser) [inline moveit_core/background_processing/src/background_processing.cpp]:
```suggestion
const rclcpp::Logger LOGGER = rclcpp::get_logger("background_processing");
```

Comment (mlautman) [reply moveit_core/background_processing/src/background_processing.cpp]: We've been down that rode and Acutronics is having issues with const loggers. I think we let it slide for now and make the change to const repo wide after the port

Comment (henningkayser) [reply moveit_core/background_processing/src/background_processing.cpp]: I thought the problem was with having static loggers?

Comment (anasarrak) [reply moveit_core/background_processing/src/background_processing.cpp]: You have the error https://github.com/ros-planning/moveit2/issues/21#issuecomment-478259602 and there is an open issue just for this https://github.com/ros2/rclcpp/issues/679 and an open [PR](https://github.com/ros2/rclcpp/pull/680)

**Pull request merged**